doc/.wordlist.txt

```diff
@@ -323,10 +331,10 @@ UFW
 UI
 UID
 UIDs
+uncomment
 unconfigured
 unevictable
 unixgram
-uncomment
 unmanaged
 unmount
 unmounting
```

Comment:
This PR adds the following entries to the word list:
- api
- blocksize
- config
- filesystem
- iSCSI
- refquota
- truenas
- TrueNAS

Of those, the only ones that I think are reasonable to add are:
- iSCSI
- TrueNAS

The others should either use their long form equivalents ("block size", "file system", "configuration") or if they refer to a specific key name or internal construct, they should be marked as such like `refquota`.